lib/iex/lib/iex/h_elixir.ex

```diff
@@ -0,0 +1,143 @@
+defmodule IEx.H_Elixir do
```

Comment:
Ok, I'm thinking of changing them to 

IEx.DocHelp.Elixir, IEx.DocHelp.ErlangStub

or 

IEx.DocExilir , IEx.DocErlangStub

I'm thinking the first might be useful to abstract some common functionality between the doc helpers. This might address some of the other comments.